segmentation/utils/synthetic_spatial_datagen.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import xarray as xr
+import pandas as pd
+import skimage.measure
+import scipy
+import os
+import xarray as xr
+import copy
+
+from segmentation.utils import spatial_analysis_utils as sau
+from scipy.spatial.distance import cdist
+from skimage.measure import label
+
+
+def direct_init_dist_matrix(num_A=100, num_B=100, num_C=100,
+                            ab_dist_mean=100, ab_dist_var=1, ac_dist_mean=20, ac_dist_var=1,
+                            distr_AB={'mean': 100, 'var': 1}, distr_AC={'mean': 20, 'var': 1},
+                            seed=None):
+    """
+    This function will return a random dist matrix such that the distance between cells
+    of types A and B are overall larger than the distance between cells of types A and C
+
+    Each row and column representing a cell.
+    We generate the points using Gaussian distributions
+    Ideally, the parameters for A to B distances will be set such that they produce a lower range of values
+    than A to C distances.
+
+    Will return a random distance matrix as well as the dictionary of associated cell: label IDs
```

Comment:
Lets update this with Args, Returns format, and then update the Returns to match what the function is now returning